exaudfclient/base/python_repository.bzl

```
@@ -41,8 +41,10 @@ def _python_local_repository_impl(repository_ctx):
     python_version_env_var = repository_ctx.name.upper() + "_VERSION"
     if python_version_env_var in repository_ctx.os.environ:
         version = repository_ctx.os.environ[python_version_env_var]
-        if repository_ctx.name == "python3" and version.startswith("2") or repository_ctx.name == "python2" and version.startswith("3"):
-            fail("Wrong python version specified in environment variable %s, exppected %s, got %s"%(python_version_env_var,repository_ctx.name,version))
+        if repository_ctx.name == "python3" and version.startswith("2"):
+            fail("Wrong python version specified in environment variable %s, expected %s, got %s"%(python_version_env_var,repository_ctx.name,version))
```

Actually this error message does not make much sense, an example would be:
```
Wrong python version specified in environment variable my_var, expected 'python3', got `2.7`. 
```
Maybe better would be
```suggestion
            fail("Wrong python version specified in environment variable %s, got binary name '%s', but version number '%s'"%(python_version_env_var,repository_ctx.name,version))
```